Loop.xcodeproj/project.pbxproj

```diff
@@ -3046,6 +3093,26 @@
 /* End PBXResourcesBuildPhase section */
 
 /* Begin PBXShellScriptBuildPhase section */
+		1D72136D2527D4AA009CFEA3 /* Upload symbols to Crashlytics */ = {
```

Comment:
Does this get run with every build?  People that are building Loop and not using Crashlytics should not have their symbols uploaded. I also don't see any authentication credentials being used here. Are the symbols uploaded via an open end point?

Reply:
Yes, good point.  I'll work on changing this to only be uploaded by CI.  Stay tuned.